File: DPBO-praktikum/1/Python/Mahasiswa.py
class Mahasiswa:

    __name=""
    __nim=""
    __prodi=""
    __fakultas=""

    def __init__(self, name="", nim="", prodi="", fakultas=""):
        self.__name=name
        self.__nim=nim
        self.__prodi=prodi
        self.__fakultas=fakultas

    def get_name(self):
        return self.__name
    
    def get_nim(self):
        return self.__nim
    
    def get_prodi(self):
        return self.__prodi
    
    def get_fakultas(self):
        return self.__fakultas

    def set_name(self, name):
        self.__name=name

    def set_nim(self, nim):
        self.__nim=nim

    def set_prodi(self, prodi):
        self.__prodi=prodi

    def set_fakultas(self, fakultas):
        self.__fakultas=fakultas

File: DPBO-praktikum/1/CPP/Mahasiswa.cpp
//library yang dipakai
#include<iostream>
#include<string>
//standar library
using namespace std;

class Mahasiswa
{
    //variabel private
private:
    string name;
    string nim;
    string prodi;
    string fakultas;

public:
    //constructor
    Mahasiswa(){
        this->name="";
        this->nim="";
        this->prodi="";
        this->fakultas="";
    }
    // construktor
    Mahasiswa(string name, string nim, string prodi, string fakultas)
    {
        this->name = name;
        this->nim = nim;
        this->prodi = prodi;
        this->fakultas = fakultas;
    }

    string get_name() {
        return this->name;
    }

    string get_nim() {
        return this->nim;
    }

    string get_prodi() {
        return this->prodi;
    }

    string get_fakultas() {
        return this->fakultas;
    }

    void set_name(string name) {
        this->name = name;
    }

    void set_nim(string nim) {
        this->nim = nim;
    }

    void set_prodi(string prodi) {
        this->prodi = prodi;
    }

    void set_fakultas(string fakultas) {
        this->fakultas = fakultas;
    }
    //destructor
    ~Mahasiswa()
    {

    }
};
